### PR TITLE
cpu/sam0: fix or add missing EXTI on GPIO

### PR DESCRIPTION
**Pull request by dylad**
Add one missing EXTI for SAML21 and add or fix several EXTI for SAMD21

I tested the new EXTI for saml21 but I don't have samr21-xpro board so if someone could test it... It would be great !

### CODE REVIEW

File: cpu/samd21/periph/gpio.c [hunk 1]
```diff
@@ -41,10 +41,17 @@
...
+    { 0,  1,  2,  3,  4,  5,  6,  7, -1,  9, 10, 11, 12, 13, 14, 15,
+      0,  1,  2,  3,  4,  5,  6,  7, 12, 13, -1, 15,  8, -1, 10, 11},
+    { 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15,
+      0,  1, -1, -1, -1, -1,  6,  7, -1, -1, -1, -1, -1, -1, 14, 15},
+#elif CPU_MODEL_SAMR21G18A
     {-1,  1, -1, -1,  4,  5,  6,  7, -1,  9, 10, 11, 12, 13, 14, 15,
      -1,  1,  2,  3, -1, -1,  6,  7, 12, 13, -1, 15,  8, -1, 10, 11},
     { 0, -1,  2,  3, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1,
-      0,  1, -1, -1, -1, -1,  6,  7, -1, -1, -1, -1,  8, -1, -1, -1},
+      0,  1, -1, -1, -1, -1,  6, 7,  -1, -1, -1, -1,  8, -1, -1, -1},
```
Comment (haukepetersen) [inline cpu/samd21/periph/gpio.c]: this line just changes the position of one ' ' char?!

Comment (dylad) [inline cpu/samd21/periph/gpio.c]: Well yes :)
just to align things.
BTW, what do you think of #ifdef CPU_MODEL_SAMD21J18A and #elif CPU_MODEL_SAMR21G18A ?

Comment (dylad) [inline cpu/samd21/periph/gpio.c]: Oh wait my bad !
This is a mistake, I'll change that.

**Comment by dylad** [comment]
Sorry about that @haukepetersen 
I fixed it.

**Comment by miri64** [comment]
7c6c11c either needs to be squashed or properly prefixed. The way it is now the CI recognizes it as a commit, that needs to be squashed.

**Comment by dylad** [comment]
@miri64 I'll squash.

**Comment by dylad** [comment]
Squashed, Murdock is happy :)

**Comment by aabadie** [comment]
Then go ! Thanks @dylad

**Pull request merged**